### PR TITLE
swe l1a validation data

### LINKED ISSUE #1036: L1A Science Data validation

### PR DESCRIPTION
**Pull request by tech3371**
# Change Summary
closes #1036 

## Overview
Validate SWE L1A data

## Deleted Files
- imap_processing/tests/swe/decompressed/
   - deleted old validation data

## Updated Files
- imap_processing/tests/swe/conftest.py
   - updated test to read validation data
- imap_processing/tests/swe/test_swe_l1a_science.py
   - update tests

### CODE REVIEW

File: imap_processing/tests/swe/l1_validation/swe_l0_unpacked-data_20240510_v001_VALIDATION_L1A.dat
Comment (tmplummer) [inline imap_processing/tests/swe/l1_validation/swe_l0_unpacked-data_20240510_v001_VALIDATION_L1A.dat:1]: Do you feel good about the test dataset? I was surprised at how many repeat values are in the validation data and it made me wonder how thoroughly it is exercising the decompression. Just a thought.

**Review by tmplummer — APPROVED** [review]
This looks good to me. Just had the one question about the test data that you have been provided.

File: imap_processing/tests/swe/l1_validation/swe_l0_unpacked-data_20240510_v001_VALIDATION_L1A.dat
Comment (tech3371) [reply imap_processing/tests/swe/l1_validation/swe_l0_unpacked-data_20240510_v001_VALIDATION_L1A.dat:1]: Ruth seems to be aware that this is not as good as they like but this was a first good step. Up until this point, many of us didn't have idea of how we would validate each other. Now, this work laid good ground for better future validation when they have better data.

**Review by sdhoyt — APPROVED** [review]
Looks good!

Edit: Looks like I opened to this page before you merged and didn't refresh before finishing the review lol

**Pull request merged**